From ea99823376006a2caa67230b48cb5fe13f7dadbd Mon Sep 17 00:00:00 2001
From: Lukas Czerner <lczerner@redhat.com>
Date: Thu, 23 Oct 2014 09:54:24 -0400
Subject: [fs] jbd2: mark file-local functions as static

Message-id: <1414058125-4183-111-git-send-email-lczerner@redhat.com>
Patchwork-id: 98198
O-Subject: [RHEL 7.1 PATCH v2 110/171] jbd2: mark file-local functions as static
Bugzilla: 1150139
RH-Acked-by: Eric Sandeen <sandeen@redhat.com>

From: Rashika Kheria <rashika.kheria@gmail.com>

BZ 1150139
https://bugzilla.redhat.com/show_bug.cgi?id=1150139

BREW 8142710
https://brewweb.devel.redhat.com/taskinfo?taskID=8142710

Upstream commit 7747e6d028b891f3bd02d93295d80f230ba43f6a

 Mark functions as static in jbd2/journal.c because they are not used
 outside this file.

 This eliminates the following warning in jbd2/journal.c:
 fs/jbd2/journal.c:125:5: warning: no previous prototype for 'jbd2_verify_csum_type' [-Wmissing-prototypes]
 fs/jbd2/journal.c:146:5: warning: no previous prototype for 'jbd2_superblock_csum_verify' [-Wmissing-prototypes]
 fs/jbd2/journal.c:154:6: warning: no previous prototype for 'jbd2_superblock_csum_set' [-Wmissing-prototypes]

 Signed-off-by: Rashika Kheria <rashika.kheria@gmail.com>
 Signed-off-by: "Theodore Ts'o" <tytso@mit.edu>
 Reviewed-by: Josh Triplett <josh@joshtriplett.org>
 Reviewed-by: Darrick J. Wong <darrick.wong@oracle.com>
 Signed-off-by: Lukas Czerner <lczerner@redhat.com>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/jbd2/journal.c b/fs/jbd2/journal.c
index 5fa344a..244b6f6 100644
--- a/fs/jbd2/journal.c
+++ b/fs/jbd2/journal.c
@@ -122,7 +122,7 @@ EXPORT_SYMBOL(__jbd2_debug);
 #endif
 
 /* Checksumming functions */
-int jbd2_verify_csum_type(journal_t *j, journal_superblock_t *sb)
+static int jbd2_verify_csum_type(journal_t *j, journal_superblock_t *sb)
 {
  if (!JBD2_HAS_INCOMPAT_FEATURE(j, JBD2_FEATURE_INCOMPAT_CSUM_V2))
   return 1;
@@ -143,7 +143,7 @@ static __be32 jbd2_superblock_csum(journal_t *j, journal_superblock_t *sb)
  return cpu_to_be32(csum);
 }
 
-int jbd2_superblock_csum_verify(journal_t *j, journal_superblock_t *sb)
+static int jbd2_superblock_csum_verify(journal_t *j, journal_superblock_t *sb)
 {
  if (!JBD2_HAS_INCOMPAT_FEATURE(j, JBD2_FEATURE_INCOMPAT_CSUM_V2))
   return 1;
@@ -151,7 +151,7 @@ int jbd2_superblock_csum_verify(journal_t *j, journal_superblock_t *sb)
  return sb->s_checksum == jbd2_superblock_csum(j, sb);
 }
 
-void jbd2_superblock_csum_set(journal_t *j, journal_superblock_t *sb)
+static void jbd2_superblock_csum_set(journal_t *j, journal_superblock_t *sb)
 {
  if (!JBD2_HAS_INCOMPAT_FEATURE(j, JBD2_FEATURE_INCOMPAT_CSUM_V2))
   return;
-- 
1.7.1